benchmark/README.md

```diff
@@ -32,7 +32,7 @@ directory, see [the guide on benchmarks](../doc/guides/writing-and-running-bench
 | module          | Benchmarks for the `module` subsystem.                                                                           |
 | net             | Benchmarks for the `net` subsystem.                                                                              |
 | path            | Benchmarks for the `path` subsystem.                                                                             |
-| perf_hooks      | Benchmarks for the `perf_hooks` subsystem.                                                                       |
+| perf\_hooks      | Benchmarks for the `perf_hooks` subsystem.                                                                       |
```

Comment:
```suggestion
| perf\_hooks     | Benchmarks for the `perf_hooks` subsystem.                                                                       |
```

Reply:
Fixed. Thanks. I'm pretty sure that consistent alignment of pipes in the text representation of tables is something that can be configured in the formatter, so I'll look into doing that.

Reply:
Turns out it's the default with `remark-gfm` which we use in the linter, so this will happen automatically when we start using the linter for the formatter. I was omitting `gfm` when running `remark` on the command line, but I won't do that in subsequent PRs. Thanks again!